README.md

```diff
@@ -463,7 +464,7 @@ Use [JITDB's prepare](https://github.com/ssb-ngi-pointer/jitdb/#prepareoperation
 
 Waits for the index with name `indexName` to be in sync with the main
 log and then call `cb` with no arguments. If `indexName` is not
-provided, the base index will be used. 
+provided, the base index will be used.
```

Comment:
can you add `compact` to this document under methods?

Reply:
Oh, yes!